### PR TITLE
Fix minor issues in 5.11.0 docs

### PR DESCRIPTION
**Pull request by pavinduLakshan**
$subject

## Related issues

https://github.com/wso2/product-is/issues/14113

### CODE REVIEW

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/learn/configuring-recaptcha-for-self-registration.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ You can either configure the recaptcha for a tenant, or configure it globally.
     ![self-registration-enable-recaptcha](../assets/img/using-wso2-identity-server/self-registration-enable-recaptcha.png)
 7.  You have now successfully configured reCaptcha for the self
     registration flow. Start the WSO2 Identity Server and log in to the
-    user portal using the following link:
-    <https://localhost:9443/user-portal>
+    My Account using the following link:
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-recaptcha-for-self-registration.md:36]:
```suggestion
    My Account portal using the following link:
```

File: en/docs/learn/configuring-recaptcha-for-self-registration.md [hunk 2]
```diff
@@ -140,8 +140,8 @@ You can either configure the recaptcha for a tenant, or configure it globally.
 4.  Set up reCaptcha with the WSO2 Identity Server. For instructions on
     how to do this and more information about reCaptcha, see [Setting Up
     ReCaptcha](../../learn/setting-up-recaptcha).
-5.  Start the WSO2 Identity Server and log in to the user portal
-    using the following link: `<https://localhost:9443/user-portal>`
+5.  Start the WSO2 Identity Server and log in to the My Account
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-recaptcha-for-self-registration.md:143]:
```suggestion
5.  Start the WSO2 Identity Server and log in to the My Account portal
```

File: en/docs/setup/migrating-to-5110.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ Once all the above prerequisites have been met, follow the instructions given be
         transformToSymmetric: "true"
         ```
     
-    Under each migrator's parameters, find the property value of **currentEncryptionAlgrithm** and ensure that it matches with the value of the `org.wso2.CipherTransformation` property found in the `<OLD_IS_HOME>/repository/conf/carbon.properties` file.
+    Under each migrator's parameters, find the property value of **currentEncryptionAlgorithm** and ensure that it matches with the value of the `org.wso2.CipherTransformation` property found in the `<OLD_IS_HOME>/repository/conf/carbon.properties` file.
```
Comment (mefarazath) [inline en/docs/setup/migrating-to-5110.md:231]: Can we make sure that this is the value read by the code too?

Comment (ashendes) [reply en/docs/setup/migrating-to-5110.md:231]: It is, [1].

[1] https://github.com/wso2-extensions/identity-migration-resources/blob/4e25bed0924f353f30c2d07c1680aeaf7c9b4e41/components/org.wso2.is.migration/migration-resources/migration-config.yaml#L211

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**